modules/installation-azure-limits.adoc

```diff
@@ -61,6 +61,12 @@ You can [provide an install-config](../overview.md#multiple-invocations) to
 [configure](customization.md) the installation program to use specific zones to override the defaults.
 ////
 
+|OS Disk
+|
+|
+|* VM OS disk must be a minimum of 1 TiB Premium SSD (P30) which gives 5000 IOPS / 200MBps. In Azure, disk performance is directly dependent on SSD disk sizes, so to achieve the throughput supported by `Standard_D8s_v3` and the target of 5000 IOPS, at least a P30 disk is required.
+* Host caching must be set to `ReadOnly` for low read latency and high read IOPS and throughput. The reads performed from the cache, which is present either in the VM memory or in the local SSD disk, are much faster than the reads from the data disk, which is in the blob storage.
+
```

Comment:
What are your thoughts on removing the bullets? For consistency with the rest of the table, I would recommend removing them and having these two points be separate paragraphs only.

Reply:
I wasn't very confident on this either. The reason I took a call to proceed with bullets is because they both are mutually exclusive unlike others where both paragraphs are somehow related. But yes, visually it looks more consistent without bullets.

Reply:
@codyhoag - Made the changes suggested. Can be merged.